src/app/atoms/Identicon.tsx

```diff
@@ -65,3 +81,15 @@ const Identicon: React.FC<IdenticonProps> = ({
 };
 
 export default Identicon;
+
+function estimateOptimalFontSize(hash: string) {
+  const initialsLength = Math.min(initials(hash).length, MAX_INITIALS_LENGTH);
+  if (initialsLength > 2) {
+    const n = initialsLength;
+    const multiplier = Math.sqrt(
```

Comment:
![Screenshot_2020-11-04_10-47-57](https://user-images.githubusercontent.com/10885399/98091262-d8b69b00-1e8d-11eb-8037-a4d8e6b98fda.png)
Here is Identicon with size of 100, font size of 50, which is default, and some objects with labeled sizes. 'Oro Oro Oro' token name was picked specially to provide maximal text length (in px). The goal here is to fit text's box into circle.
In general case, the total length `l_max` of such string with default font size is `32n + 4(n-1)`, where `n` is amount of characters.
Let `q` be the scale factor to apply to text to reach the goal. Therefore, we get characteristics of icon objects as on the figure below.
![figure1](https://user-images.githubusercontent.com/10885399/98093074-23d1ad80-1e90-11eb-9c60-e20e81075203.png)
According to Pythagorean theoreme, we have:
```
l_max**2 * q**2 / 4 + 36**2 * q**2 / 4 = 2500;
q**2 * (l_max**2 + 36**2) = 10000;
q = sqrt(10000 / (l_max**2 + 36**2)) = sqrt(10000 / ((32n + 4(n-1))**2 + 36**2))
```
This scale is applied by multiplying font size by this factor with floor rounding to ensure that integer value is passed to the icon function and the box still fits the circle. That's why font size will be `50 * q`.
Let's not simplify this expression to keep is as clearer as possible.